### PR TITLE
feature: allow to delete all activities from portfolio

### PR DESCRIPTION
**Pull request by x1c0**
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Fixed the targets in `proxy.conf.json` from `http://localhost:3333` to `http://0.0.0.0:3333` for local development
+- Introduced a new button to delete all activities from the portfolio activities page
```
Comment (dtslvr) [inline CHANGELOG.md:13]: Please move to the section `### Added`

File: apps/api/src/app/order/order.controller.ts [hunk 2]
```diff
@@ -62,6 +62,23 @@ export class OrderController {
     });
   }
 
+  @Delete()
```
Comment (dtslvr) [inline apps/api/src/app/order/order.controller.ts:65]: Can you move this function up?

First `@Delete()`, then `@Delete(':id')`

File: apps/client/src/app/services/data.service.ts [hunk 3]
```diff
@@ -150,6 +150,10 @@ export class DataService {
     return this.http.delete<any>(`/api/v1/order/${aId}`);
   }
 
+  public deleteAllOrders() {
```
Comment (dtslvr) [inline apps/client/src/app/services/data.service.ts:153]: Can you move this function up so the order is alphabetical?

File: libs/ui/src/lib/activities-table/activities-table.component.html [hunk 4]
```diff
@@ -53,6 +53,14 @@
       <ion-icon class="mr-2" name="calendar-clear-outline"></ion-icon>
       <span i18n>Export Drafts as ICS</span>
     </button>
+    <button
+      class="align-items-center d-flex"
+      mat-menu-item
+      (click)="onDeleteAllActivities()"
+    >
+      <ion-icon class="mr-2" name="trash-outline"></ion-icon>
+      <span i18n>Delete all activities</span>
```
Comment (dtslvr) [inline libs/ui/src/lib/activities-table/activities-table.component.html:62]: Delete all **A**ctivities

File: libs/ui/src/lib/activities-table/activities-table.component.ts [hunk 5]
```diff
@@ -52,6 +52,7 @@ export class ActivitiesTableComponent implements OnChanges, OnDestroy, OnInit {
   @Output() export = new EventEmitter<string[]>();
   @Output() exportDrafts = new EventEmitter<string[]>();
   @Output() import = new EventEmitter<void>();
+  @Output() deleteAllActivities = new EventEmitter<void>();
```
Comment (dtslvr) [inline libs/ui/src/lib/activities-table/activities-table.component.ts:55]: Can you move up: before`@Output() export...`

**Review by dtslvr — CHANGES_REQUESTED** [review]

File: apps/client/src/app/services/data.service.ts (hunk 3 above)
Comment (x1c0) [reply apps/client/src/app/services/data.service.ts:153]: there is some kind of plugin that does this automatically :) maybe worth investigating

File: libs/ui/src/lib/activities-table/activities-table.component.html (hunk 4 above)
Comment (x1c0) [reply libs/ui/src/lib/activities-table/activities-table.component.html:62]: actually in English we don't capitalize names like in German :) but i'll change it :P

**Review by dtslvr — APPROVED** [review]
Thank you for the modifications. Looks fine! 👍🏻

**Pull request merged**